app/views/ui/textarea/index.html.erb

```diff
@@ -0,0 +1,104 @@
+<div class="my-6">
+  <%= render "shared/components/back_button", back_link: ui_components_path %>
+</div>
+
+<%= h3_component(text: 'Textarea Component Demo') %>
+
+<ul class="w-1/2 my-6 flex flex-col gap-4">
+  <li class="my-6 flex flex-col gap-4">
```

Comment:
Added a component for presentation in MY PR, you can use that for presenting the demo in a uniform way
https://github.com/openvitae-tech/blackboard-lms/pull/1089/files#diff-53015b9898701e40d5f360482f6c8845c07e1a2bdfe2cdb2bda453f4b0293dcb